release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3118,6 +3118,8 @@ In the following tables, features are marked with the following statuses:
 
 * When a {product-title} {product-version} cluster is installed or configured as a postinstallation activity on a single VIOS host with virtual SCSI storage on {ibm-power-name} with multipath configured, the CoreOS nodes with multipath enabled fail to boot. This behavior is expected as only one path is available to the node. (link:https://issues.redhat.com/browse/OCPBUGS-32290[*OCPBUGS-32290*]).
 
+* When using CPU load balancing on cgroupv2, a pod can fail to be created if another pod that has access to exclusive CPUs already exists. This can happen when a pod is deleted and another one is quickly created to replace it. As a workaround, ensure that the old pod is fully terminated before attempting to create the new one. (link:https://issues.redhat.com/browse/OCPBUGS-34812[*OCPBUGS-34812*]) 
```

Comment:
Would "can fail to start" keep the same meaning? Just a suggestion to be more concise, if it works.

Reply:
@haircommander @sunilcio WDYT ^^

Reply:
yeah it's just as correct (they're both slightly wrong but for this context I think it's okay)